source/fundamentals/linq.txt

```diff
@@ -320,6 +320,71 @@ following documents:
    { "date" : ISODate("2012-12-05T00:00:00Z"), "grade" : "A", "score" : 13 }
    { "date" : ISODate("2012-05-17T00:00:00Z"), "grade" : "A", "score" : 11 }
 
+Nested Statements
++++++++++++++++++
+
+You can chain or nest ``Select`` and ``SelectMany`` statements. The examples
+in this section use the ``Guidebook`` class, which contain an array of ``Restaurant``
+objects. The following code shows an example of a document modeled by the 
+``Guidebook`` class:
+
+.. code-block:: json
+
+   { 
+      "_id": { "$oid": ... },
+      "restaurants": [
+         {
+            "_id": { ... } ,
+            "address": { ... },
+            "name": "Tov Kosher Kitchen",
+            ...
+         },
+         {
+            "_id": { ... } ,
+            "address": { ... },
+            "name": "Harriet's Kitchen",
+            ...
+         },
+         ...
+      ]
+   }
+
+The following code models the ``Guidebook`` class:
+
+.. literalinclude:: /includes/fundamentals/code-examples/linq.cs
+   :language: csharp
+   :start-after: start-guidebook-model
+   :end-before: end-guidebook-model
+
+You can nest ``SelectMany`` statements within ``SelectMany`` or ``Select``
+statements. The following example nests a ``SelectMany`` statement within a
```

Comment:
Q: can you briefly explain why you'd want to nest these / what nesting the statements does? From my understanding (I'm not super familiar with c#), it would be something like:
```suggestion
statements to process nested document fields. The following example nests a ``SelectMany`` statement within a
```